connectivity/drivers/lora/TARGET_STM32WL/STM32WL_LoRaRadio.cpp

```diff
@@ -734,26 +716,21 @@ void STM32WL_LoRaRadio::init_radio(radio_events_t *events)
 
     _radio_events = events;
 
-//    SubgRf.RxContinuous = false;
     _tx_timeout = 0;
     _rx_timeout = 0;
 
     //call to HAL_SUBGHZ_Init() for MSPInit and NVIC Radio_IRQ setting
     error_value = HAL_SUBGHZ_Init(&hsubghz);
 
-    if (error_value != HAL_OK)
-    {
-        error_handler(error_value);
-    }
-
+    MBED_ASSERT(error_value == HAL_OK);
+  
     // this is a POR sequence
     cold_start_wakeup();
 
     SUBGRF_SetTxParams(RFO_LP, 0, RADIO_RAMP_200_US);
-
-    /* ST_WORKAROUND_BEGIN: Sleep radio */
+    
     sleep();
```

Comment:
@ludoch-stm  one question: where is this function sleep defined ? Could there be a more precise name, like lora_radio_sleep because sleep is usually also used at system level and should not be called by a driver ?